ouroboros-consensus-protocol/src/Ouroboros/Consensus/Protocol/Praos/Common.hs

```diff
@@ -49,13 +49,14 @@ newtype MaxMajorProtVer = MaxMajorProtVer
 -- 1. By chain length, with longer chains always preferred.
 -- 2. If the tip of each chain was issued by the same agent, then we prefer
 --    the chain whose tip has the highest ocert issue number.
--- 3. By the leader value of the chain tip, with lower values preferred.
+-- 3. By a VRF value from the chain tip, with lower values preferred. See
+--    @pTieBreakVRFValue@ for which one is used.
 data PraosChainSelectView c = PraosChainSelectView
   { csvChainLength :: BlockNo,
     csvSlotNo      :: SlotNo,
     csvIssuer      :: SL.VKey 'SL.BlockIssuer c,
     csvIssueNo     :: Word64,
-    csvLeaderVRF   :: VRF.OutputVRF (VRF c)
+    csvTieBreakVRF :: VRF.OutputVRF (VRF c)
```

Comment:
Why do we think this is a better name than the previous one? Aren't in any case `csvChainLength` and `csvSlotNo` tie breakers? What about `csvChainTipVRF`?

Reply:
In `TPraos`, `csvLeaderVRF` is the correct name, as we actually use the leader VRF value for tie breaking. But in `Praos` (which uses the same `SelectView`, i.e. this one), we no longer use the leader VRF, so this name is incorrect in that situation.

Hence, I chose a name indicating the purpose of the VRF in the context of chain selection, as it is the final tie breaker, compared to e.g. `csvChainLength` which causes the ties in the first place. (Side remark: `csvSlotNo` is not used for anything, we could remove it.)

But we don't have to stress its role as a tie breaker in the field name, e.g. `csvChainTipVRF` also sounds reasonable, as it also (intentionally!) does not mention which exact VRF from the header is used.

Reply:
> Hence, I chose a name indicating the purpose of the VRF in the context of chain selection, as it is the final tie breaker,

I see, that makes sense. I'm ok with that name. Thanks!